### PR TITLE
Early stop bug fix

### PR DESCRIPTION
**Pull request by inventormc**
`self.early_stopping` was not set properly when using `early_stopping=True`

### CODE REVIEW

File: tune_sklearn/tune_basesearch.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def __init__(self,
         if early_stopping and self._can_early_stop():
             self.max_iters = max_iters
             if early_stopping is True:
-                early_stopping = "AsyncHyperBandScheduler"
+                self.early_stopping = "AsyncHyperBandScheduler"
             elif isinstance(early_stopping, str):
```
Comment (richardliaw) [inline tune_sklearn/tune_basesearch.py:213]: I think this doesn't work; 

the idea here was to make early_stopping = "AsyncHyperBandScheduer" so it is checked in L214.

Comment (richardliaw) [reply tune_sklearn/tune_basesearch.py:213]: otherwise, self.early_stopping is never actually set into a scheduler object

Comment (richardliaw) [inline tune_sklearn/tune_basesearch.py:213]:
```suggestion
                # Override the early_stopping variable so
                # that it is resolved appropriately in
                # the next block
                early_stopping = "AsyncHyperBandScheduler"

            # Resolve the early stopping object
            if isinstance(early_stopping, str):
```

Comment (inventormc) [reply tune_sklearn/tune_basesearch.py]: Whoops yeah you're right. Changed to address this.

**Pull request merged**